dev/com.ibm.ws.jsfContainer.2.2/resources/com/ibm/ws/jsf/container/resources/JSFContainerMessages.nlsprops

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME none
+#COMPONENTPREFIX JSFG
+#COMPONENTNAMEFOR JSFG jsf
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_VAR
+# -------------------------------------------------------------------------------------------------
+
+# JSFG0100-JSFG0199 reserved for jsfContainer-2.X features
+
+# 0: JSF provider type (will be either MOJARRA or MYFACES)
+# 1: application name
+jsf.container.init=JSFG0100I: Liberty initialized JSF integrations for JSF provider {0} on application {1}
```

Comment:
Please add a period at the end of this sentence.

Reply:
Maybe something like this? I don't really like it, but there's nothing grammatically wrong with it, and I can't think of anything better!
For JSF provider {0} on application {1}, Liberty initialized JSF integrations.